app/assets/javascripts/editor.js

```diff
@@ -280,6 +279,39 @@ function (ko, models, tools, msg, validate, owned) {
             }
         };
 
+        self.doUpdateFigureCaption = function () {
+
+            if (self.hasAbstractFigures()) {
+                var figure = self.abstract().figures()[0],
+                    data = new FormData(),
```

Comment:
This does not need to be `FormData` since its only the figure object without the actual file